src/main/java/frc/trigon/robot/RobotContainer.java

```diff
@@ -48,21 +56,39 @@ private void initComponents() {
 
     private void initCommands() {
         swerveCmd = new FieldRelativeSupplierDrive(
-                () -> -controller.getLeftY(),
-                () -> controller.getLeftX(),
+                () -> controller.getLeftY(),
+                () -> -controller.getLeftX(),
                 () -> -controller.getRightX()
         );
         collectCommand = new CollectCommand();
 
         countBallsCommand = new CountBallsCommand();
         shotsDetectorCommand = new ShotsDetectorCommand();
+
+        turnToHubCmd = new TurnToTargetCommand(limelight, 0);
+
+        primeShooterCmd = Shooter.getInstance().getPrimeShooterCommandWithDefault(
+                () -> limelight.hasTarget() ?
+                      ShootingCalculations.getShootingVelocityFromDistance(limelight.getDistanceFromHub()) :
+                      0
+        );
+        pitchCmd = Pitcher.getInstance().getPitchingCommandWithDefault(
+                () -> limelight.hasTarget() ?
+                      ShootingCalculations.getShootingAngleFromDistance(limelight.getDistanceFromHub()) :
+                      0
+        );
     }
 
     private void bindCommands() {
         Swerve.getInstance().setDefaultCommand(swerveCmd);
+        Pitcher.getInstance().setDefaultCommand(pitchCmd);
+        Shooter.getInstance().setDefaultCommand(primeShooterCmd);
```

Comment:
Wont this make them always run?

Reply:
I have a feeling that you want them to always run. do we have enough battery for that?

Reply:
Maybe